WorldBank-Transport/DRIVER
# Config Wizard #154147803

## Overview

Add configuration wizard to facilitate deployment.

## Testing Instructions

* Read documentation to see if it gives enough info and is helpful and well-written
* Ensure you have Python 3.6 and pip installed (outside VM)
* Run `./scripts/setup` outside VM and fill in values at prompts.
* Check to see that `deployment/ansible/group_vars/production` and `deployment/ansible/inventory/production` were generated and look correct.
* Check that new files are ignored

# PivotalTracker

https://www.pivotaltracker.com/story/show/154147803
